From 04f592a096f53a2abf1905fdbdbb359d26a6eb51 Mon Sep 17 00:00:00 2001
From: Sam Yang <r52096@freescale.com>
Date: Tue, 16 Mar 2010 17:05:26 +0800
Subject: [PATCH] ENGR00121615 Remove duplicate definition of fec_platform_data in header file

new definition ready in include/linux/fec.h.
remove old mxc definition from  arch/arm/plat-mxc/include/mach/mxc.h

Signed-off-by: Sam Yang <r52096@freescale.com>
---
 arch/arm/plat-mxc/include/mach/mxc.h |    6 ------
 1 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/mxc.h b/arch/arm/plat-mxc/include/mach/mxc.h
index df0575d..efe4323 100644
--- a/arch/arm/plat-mxc/include/mach/mxc.h
+++ b/arch/arm/plat-mxc/include/mach/mxc.h
@@ -447,12 +447,6 @@ struct tve_platform_data {
 	char *dig_reg;
 };
 
-struct fec_platform_data {
-	int (*init) (void);
-	int (*uninit) (void);
-	struct regulator *vddio_reg;
-};
-
 struct mxc_vpu_platform_data {
 	void (*reset) (void);
 };
-- 
1.5.4.4

